Bundler/Viewer/Unity/Assets/Scripts/Network/Scripts/NetworkAnchorHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class NetworkAnchorHelper : MonoBehaviour
+{
+    public GameObject NetworkAnchorManagerPrefab;
+    public GameObject NetworkAnchorPlayerPrefab;
+
+    /// <summary>
+    /// Track the anchor manager spawned by this class. Only one manager per session.
+    /// </summary>
+    private static GameObject spawnedNetworkAnchorManager;
+
+    /// <summary>
+    /// Prepare the given network manager to host an anchor manager and anchor players
+    /// </summary>
+    public void PrepareNetworkManager(NetworkManager manager)
+    {
+        if (manager == null)
+        {
+            return;
+        }
+
+        if (NetworkAnchorManagerPrefab != null)
```

Comment:
is it ever ok for these to be null? if so, no need to check

Reply:
"Pending" These should never to null.  What's the best way to handle this in Unity.  Should I just throw at runtime.  Or is there a way to cause of build failure in Unity?

Reply:
good questions. i'm not quite sure how to cause a build failure. i believe throwing at runtime will likely be eaten by unity's default exception handler. I would add a Debug.LogError for now.